sensing/preprocessor/pointcloud/livox/livox_tag_filter/README.md

```diff
@@ -0,0 +1,117 @@
+# livox_tag_filter
+
+## Purpose
+
+The `livox_tag_filter` is a node that removes noise from pointcloud by using the following tags:
+
+- Point property based on spatial position
+- Point property based on intensity
+- Return number
+
+## Inner-workings / Algorithms
+
+<!-- Write how this package works. Flowcharts and figures are great. Add sub-sections as you like.
+
+Example:
+  ### Flowcharts
+
+  ...(PlantUML or something)
+
+  ### State Transitions
+
+  ...(PlantUML or something)
+
+  ### How to filter target obstacles
+
+  ...
+
+  ### How to optimize trajectory
+
+  ...
+-->
```

Comment:
@satoshi-ota FYI you can remove these template comments.

Reply:
OK :+1: I have left these templates for those who want to add more information about the package details in the documentation.

Reply:
I see, but it can be included in the common guidelines of design documents.
I'll prepare them for Autoware Core in the near future!

Reply:
OK :+1: I'll remove these template comments from documents. (I have left templetes comment in several pkgs.)